webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -3201,6 +3201,91 @@ const AdminDefinition: AdminDefinitionType = {
                     ],
                 },
             },
+            wrangler: {
+                url: 'site_config/wrangler',
+                title: t('admin.sidebar.wrangler'),
+                title_default: 'Wrangler',
```

Comment:
Can we update the admin settings to **not** use Wrangler to avoid confusion with the wrangler plugin here?

Reply:
I've replaced all user-facing occurrences of "Wrangler" with "Move Thread"